drivers/focuser/sestosenso2.cpp

```diff
@@ -99,6 +99,22 @@ bool SestoSenso2::initProperties()
     IUFillNumberVector(&SpeedNP, SpeedN, 1, getDeviceName(), "FOCUS_SPEED", "Motor Speed", MAIN_CONTROL_TAB, IP_RO, 0,
                        IPS_IDLE);
 
+    // Focuser backlash
+    IUFillNumber(&BacklashN[0], "BACKLASH", "Backlash", "Enter backlash ticks value.", -50000, 50000, 1000, 0);
```

Comment:
No need to define this, just set FOCUSER_HAS_BACKLASH and implement the backlash function from the FocuserInterface